.travis/package-osx.sh

```diff
@@ -3,5 +3,8 @@
 mkdir -p $HOME/artifacts
 
 cd $TRAVIS_BUILD_DIR/BUILD/Install
-zip -q -r --symlinks $HOME/artifacts/SC-$TRAVIS_COMMIT.zip SuperCollider
-cp $HOME/artifacts/SC-$TRAVIS_COMMIT.zip $HOME/SuperCollider-$VERSION_TO_BUILD-macOS.zip
+ls -a
+zip -q -r --symlinks $HOME/artifacts/$S3_ARTIFACT_NAME.zip SuperCollider
+ls -a $HOME/artifacts
+cp $HOME/artifacts/$S3_ARTIFACT_NAME.zip $HOME/$GITHUB_ARTIFACT_NAME.zip
+ls -a $HOME
```

Comment:
did you leave these ls's here intentionally? i don't mind them, just wondering

Reply:
nope! that's the only debug I overlooked. I kind of like this output. But I should remove that, right?

Reply:
maybe just comment it out, it could still be helpful for debugging in the future

Reply:
Sorry, didn't see this on time before rebasing. Should I add these back?

Reply:
nah it's fine, thank you for noticing though